clients/filesystem-fuse/src/main.rs

```diff
@@ -16,18 +16,32 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+use fuse3::Errno;
+use gvfs_fuse::config::AppConfig;
 use gvfs_fuse::{gvfs_mount, gvfs_unmount};
-use log::info;
+use log::{error, info};
 use tokio::signal;
 
 #[tokio::main]
 async fn main() -> fuse3::Result<()> {
     tracing_subscriber::fmt().init();
-    tokio::spawn(async { gvfs_mount("gvfs").await });
+
+    let config = AppConfig::from_file(Some("conf/gvfs_fuse.toml)"));
```

Comment:
How can we ensure this relative path is always correct?

Reply:
That's not implemented. we need to read config file path from app arguments.